docs/src/appendix/cheatcodes/tx_info.md

```diff
@@ -46,10 +46,22 @@ struct TxInfoMock {
 }
 ```
 
-Returns a default object initialized with Option::None for each field
-Useful for setting only a few of fields instead of all of them
+### `starknet::info::v2::ResourceBounds`
+```
+pub struct ResourceBounds {
+    resource: felt252,
+    max_amount: u64,
+    max_price_per_unit: u128,
+}
+```
+A struct responsible for setting the resource bounds, used in `TxInfoMock`.
+
+## `TxInfoMockTrait`
 ```
 trait TxInfoMockTrait {
     fn default() -> TxInfoMock;
 }
-```
\ No newline at end of file
+```
+
+Returns a default object initialized with Option::None for each field
+Useful for setting only a few of fields instead of all of them
```

Comment:
```suggestion
Returns a default object initialized with `Option::None` for each field.
Useful for setting only a few of the fields instead of all of them.
```